crates/red_knot_python_semantic/src/types.rs

```diff
@@ -899,7 +899,7 @@ impl<'db> Type<'db> {
                 Type::SubclassOf(_),
             ) => true,
             (Type::SubclassOf(_), _) | (_, Type::SubclassOf(_)) => {
-                // TODO: Once we have support for final classes, we can determine disjointness in some cases
+                // TODO: Once we have support for final classes, we can determine disjointedness in some cases
```

Comment:
I think the existing word is better here

https://en.wiktionary.org/wiki/disjointness

https://en.wiktionary.org/wiki/disjointedness